lib/binaries/binary.ts

```diff
@@ -81,14 +81,20 @@ export abstract class Binary {
   /**
    * Gets the url to download the file set by the version. This will use the XML if available.
    * If not, it will download from an existing url.
+   *
+   * @param {string} version The version we are looking for. This could also be 'latest'.
+   * @param {opt_proxy} string Option to get proxy URL.
+   * @param {opt_ignoreSSL} boolean Option to ignore SSL.
    */
-  getUrl(version?: string): Promise<BinaryUrl> {
+  getUrl(version?: string, opt_proxy?: string, opt_ignoreSSL?: boolean): Promise<BinaryUrl> {
+    this.opt_proxy = opt_proxy || this.opt_proxy;
```

Comment:
As this currently stands, passing `opt_proxy` once will change the default value for all future calls.  Is that the desired behavior?